test/webpack.build.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs-extra');
```

Comment:
`global-require` should not be disabled by default as it's frowned upon in general.

In the certain cases where it's necessary, it should be disabled inline on a case by case basis.